opg-elastictrim/docker/opt/elastictrim.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+if [[ -z "${OPG_ENV}" ]]; then
+  echo "OPG_ENV environment variable not defined. Exiting."  
+  exit
+fi
+
+es_address=localhost
+bucket_name="sirius-elastic-backup-${OPG_ENV}"
+archive_days_limit=${ELASTICTRIM_DAYS_OLD:-12}
+nobackup=true
+nodelete=true
```

Comment:
I want to intentionally deploy the whole setup in dev and prod where this does nothing to begin with.